modules/view-kubernetes-secrets.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-cluster-configuration.adoc
+:_module-type: PROCEDURE
+[id="view-kubernetes-secrets_{context}"]
+= Viewing Kubernetes secrets
+
+[role="_abstract"]
+View Kubernetes secrets in use in your environment and identify deployments using those secrets.
```

Comment:
Does deployments refer to clusters? If so, you could change "deployments" to "clusters" in this line to make it more explicit.